README.md

```diff
@@ -41,7 +41,7 @@ describe('audit example', () => {
     await page.goto('https://angular.io/');
 
     await playAudit({
-      page: page,
+      page: page, // or pass url as string directly
```

Comment:
1. can you please add a section of details telling what are the advantages of passing the URL directly?
2. Can we have both the options page and URL? we can have either or both. It will help us in future to achieve different actions based on the `page` options. 


wdyt?

Reply:
1. Will do, I can add an example from the original issue.
2. I can make the `page` optional and add `url`. We can throw an exception in case neither page nor url were provided

Reply:
Sounds good to me.